src/components/SuggestionsGenerator/index.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from '@wordpress/element';
+import moduleAI from '@newfold-labs/wp-module-ai';
+import apiFetch from '@wordpress/api-fetch';
+import { ReactComponent as RegenerateIcon } from '../../icons/regenerate-icon.svg';
+
+import SuggestionsList from './suggestionList';
+const SuggestionsGenerator = (props) => {
+
+	const [aiResults, setAIResults] = useState([]);
+	const siteDescInput = document.querySelector(props.targetSelector) ? document.querySelector(props.targetSelector).value : null;
+
+	const getAIResult = async (siteTitle, siteUrl, siteDesc) => {
```

Comment:
This should go in the js utility and be used here, we are not calling the ai service directly.

Reply:
const result = await moduleAI.search.getSearchResult(
					userPrompt,
					'descgenerator'
				);
				
this is calling from the js utility only?